src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -1626,23 +1759,35 @@ public C(string s)
             throw new ArgumentNullException(nameof(s));
     }
 }",
-    parameters: new TestParameters(options:
-        Option(CSharpCodeStyleOptions.PreferBraces, new CodeStyleOption2<PreferBracesPreference>((PreferBracesPreference)preferBraces, NotificationOption2.Silent))));
+                Options =
+                {
+                    { CSharpCodeStyleOptions.PreferBraces, new CodeStyleOption2<PreferBracesPreference>((PreferBracesPreference)preferBraces, NotificationOption2.Silent) }
+                }
+            }.RunAsync();
         }
 
         [WorkItem(19956, "https://github.com/dotnet/roslyn/issues/19956")]
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
         public async Task TestNoBlock()
         {
-            await TestInRegularAndScript1Async(
-@"
+            await new VerifyCS.Test
+            {
+                TestCode = @"
 using System;
 
 class C
 {
     public C(string s[||])
 }",
-@"
+                ExpectedDiagnostics = {
+                    // /0/Test0.cs(6,12): error CS0501: 'C.C(string)' must declare a body because it is not marked abstract, extern, or partial
+                    DiagnosticResult.CompilerError("CS0501").WithLocation(6, 12).WithArguments("C.C(string)"),
+                    // /0/Test0.cs(6,23): error CS1002: ; expected
+                    DiagnosticResult.CompilerError("CS1002").WithLocation(6, 23),
```

Comment:
💡 Since the test intentionally has erroneous code, you can use `{|CS0501:...|}` and `{|CS1002:...|}` to mark these directly in code if that's easier.